src/index.js

```diff
@@ -0,0 +1,175 @@
+//application states
+const state = {
+  sky: 'sunny',
+  temperature: 0,
+  city: '',
+};
```

Comment:
Instead of having a hard coded default sky state, how might you refactor this so the logic is more data driven? The sky should really be displayed based on the call to the api